src/index.js

```diff
@@ -106,6 +106,10 @@ class Dropzone extends React.Component {
     this.isFileDialogActive = false;
 
     fileList.forEach((file) => {
+      if (!(file instanceof window.File)) {
```

Comment:
What about the case there `window.File` is `undefined`? Also, this will probably break SSR implementations.